.travis.yml

```diff
@@ -6,65 +6,7 @@ php:
 # http://docs.travis-ci.com/user/workers/container-based-infrastructure/
 sudo: false
 
-matrix:
-  include:
-    # Lint checks for PHP code and composer.json  -  removed until we figure out what to do with it
-    #- php: 5.6
-    #  env: VARIA=true
-    #  install:
-    #   - phpenv rehash
-    #   - composer self-update
-    #   - composer install
-    #  script:
-    #   - bash .scripts/travis/check_commit_msgs.sh
-    #   - vendor/bin/phpcs --standard=vendor/elgg/sniffs/elgg.xml --warning-severity=0 --ignore=*/tests/*,*/upgrades/*,*/deprecated* engine/classes engine/lib
-    #   - composer validate
-       
-    # Build and test javascript
-    - php: 5.6
-      env: VARIA=true
-      install:
-       - npm install
-      script:
-       - npm test
-      
-    # Build docs and check for errors
-    # Flags used here, not in `make html`:
-    #  -n   Run in nit-picky mode. Currently, this generates warnings for all missing references.
-    #  -W   Turn warnings into errors. This means that the build stops at the first warning and sphinx-build exits with exit status 1.
-    - php: 5.6
-      env: VARIA=true
-      install: 
-       - pip install --user "Sphinx==1.3.1"
-       - pip install --user "sphinx-intl"
-       - export PATH=$PATH:$HOME/.local/bin
-      script:
-       - sphinx-build -b html -nW docs docs/_build/html
-       - sphinx-build -b latex -nW docs docs/_build/latex
-       - sphinx-intl build --locale-dir=docs/locale/
-       - sphinx-build -b html -D language=es -n docs docs/_build/html
-
-    # End to end tests
-    - php: 5.6
```

Comment:
that's it, the one line for the comment formatting can go but everything else is being used.